spec/artsy-eventstream/artsy/publisher_spec.rb

```diff
@@ -29,7 +29,7 @@
       allow(event).to receive(:verb).and_return(nil)
       expect { Artsy::EventService::Publisher.publish(topic: 'test', event: event) }.to raise_error 'Event missing topic or verb.'
     end
-    it 'calls publish on the exchange with proper data' do
+    it 'uses verb as routing key when calling publish on the exchange without passing routing_key' do
```

Comment:
i think it does in the case of someone listening on all routing_keys (`*`) and want to do different things based on the event's verb as opposed to open different connections for different routing key. An example would be Pulse where we listen on [all](https://github.com/artsy/pulse/blob/master/app/workers/rabbit_invoices_worker.rb#L6) events but then we do [this](https://github.com/artsy/pulse/blob/master/app/workers/rabbit_invoices_worker.rb#L10).